frontend/app-development/layout/PageLayout.tsx

```diff
@@ -21,7 +21,7 @@ export const PageLayout = (): React.ReactNode => {
 
   const { data: orgs, isPending: orgsPending } = useOrgListQuery();
   const { data: repository } = useRepoMetadataQuery(org, app);
-  const repoOwnerIsOrg = !orgsPending && Object.keys(orgs).includes(repository?.owner.login);
+  const repoOwnerIsOrg = !orgsPending && Object.keys(orgs).includes(repository?.owner?.login);
```

Comment:
This was a small fix because an error boundary kicked in after the session was staled.